src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -489,7 +492,7 @@ public bool TryGetInfoFromPath(string path, bool isTestMode, out CustomNodeInfo
                 nodeGraph.Notes,
                 workspaceInfo.X,
                 workspaceInfo.Y,
-                functionId, workspaceInfo.FileName);
+                functionId, elementResolver, workspaceInfo.FileName);
             
```

Comment:
The reason I'm not instantiating `ElementResolver` property inside the `WorkspaceModel` ctor is because it needs to be passed in from outside such as in this case. In my subsequent PR, `ElementResolver` will be obtained from `LoadGraphFromXml` with a call like: `var nodeGraph = NodeGraph.LoadGraphFromXml(xmlDoc, nodeFactory, out elementResolver);`